docs/user/guides/redirects.rst

```diff
@@ -0,0 +1,107 @@
+How to setup URL redirects for documentation projects
+=====================================================
+
+In this guide,
+you will learn the steps necessary to configure your Read the Docs project for redirecting visitors from one location to another.
+
+These user-defined redirects are handled at the HTTP-level,
+which means that a reader will not notice anything in their browser.
+A user visiting the *old URL* will automatically end up at the *new URL*.
+
+.. seealso::
+
+   :doc:`/automatic-redirects`
+     The need for a redirect often comes from external links to your documentation.
+     Read more about handling links in this explanation of best practices.
+
+   :doc:`/user-defined-redirects`
+     If you want to know more about our implementation of redirects,
+     you can look up more examples in our reference before continuing with the how-to.
+
+Setting up a redirect
+---------------------
+
+Redirects are configured in the project dashboard,
+go to :menuselection:`Admin > Redirects`.
+
+.. figure:: /img/screenshot_redirects.png
+   :alt: Screenshot of the Redirect admin page
+   :scale: 50%
+   :align: center
+
+   Navigate to the :guilabel:`Redirect` page and you will see an overview of all active rules.
+   In this guide, we go through how to use the :guilabel:`Add Redirect` function.
+
+After navigating to :guilabel:`Add Redirect`,
+you need to select a :guilabel:`Redirect Type`.
+This is where things get a bit more complicated you need to fill in specific information according to that choice.
+
+Choosing a :guilabel:`Redirect Type`
+------------------------------------
+
+There are different types of redirect rules targeting different needs.
+For each choice in :guilabel:`Redirect Type`,
+you can mark the choice in order to experiment with choices in order to **preview** the final rule generated.
+
+Prefix redirect
+  This option is often relevant when moving a project from a former host to Read the Docs.
+  In this case, often URL paths hierarchies are redirected.
+
+  Read more about this option in :ref:`user-defined-redirects:Prefix redirects`
+
+Page redirect
+  With this option,
+  you can specify a page in your documentation to redirect elsewhere.
+  The rule triggers no matter the version of your documentation that the user is visiting.
+  This rule can also redirect to another website.
```

Comment:
This is from the original reference. However, the preview function doesn't reflect this. Either that's a bug or the information in the reference was incorrect. Will need to check this...

Reply:
It checks out, so it's a bug in the UI that I already reported.